djconnectwise/tests/test_api.py

```diff
@@ -234,24 +236,13 @@ def setUp(self):
 
     @responses.activate
     def test_post_hosted_tab(self):
-        body = {
-            'screenId': fixtures.API_HOSTED_SCREENS['row_values'][3],
-            'description': 'A new hosted tab',
-            'url': 'https://example.com',
-            'type': 'Tab'
-        }
-
-        method_name = 'djconnectwise.api.ConnectWiseAPIClient.request'
-        mock_call, _patch = mk.create_mock_call(
```

Comment:
`create_mock_call` does not clean up after itself, so use of mocks in tests were affecting other tests. :(

I changed to use with `with` context manager, which guarantees it cleans up after itself.